.github/workflows/shared-ui-npm-publish.yml

```diff
@@ -0,0 +1,60 @@
+name: Release step 3 - Publish to NPM
+
+on:
+  # Runs on every push to main
```

Comment:
```suggestion
```
i feel the code itself is pretty concise to tell it's running on release publishing, yet the comment is actually incorrect :)

Reply:
true, i've removed this and other self-explanatory comments